docs/source/install.rst

```diff
@@ -48,12 +52,13 @@ the binaries and images.
 
 .. note:: If you want a specific release, pass a version identifier for Fabric,
           Fabric-ca and thirdparty Docker images.
-          The command below demonstrates how to download **Fabric v2.0.0 Beta release v2.0.0-beta**
+          The command below demonstrates how to download the latest production releases -
+          **Fabric v2.0.0** and **Fabric CA v1.4.4**
```

Comment:
Still feel the Fabric CA should release a v2.0.0 too, if it will support fabric v2.0.0. Otherwise, easy to confuse during usage.

Reply:
Thanks Baohua, understand the concern, however in prior contributor meeting the maintainers decided not to keep releasing Fabric CA concurrently, as we don't expect changes each quarter.